doc/book/box/data_model.rst

```diff
@@ -13,53 +13,74 @@ then your test database now looks like this:
 
 .. image:: data_model.png
 
-.. _index-box_space:
+.. _index-box_tuple:
 
 --------------------------------------------------------------------------------
-Spaces
+Tuples
 --------------------------------------------------------------------------------
 
-A **space** -- 'tester' in our example -- is a container.
+Tarantool operates data in the form of tuples.
 
-When Tarantool is being used to store data, there is always at least one space.
-Each space has a unique **name** specified by the user.
-Besides, each space has a unique **numeric identifier** which can be specified by
-the user, but usually is assigned automatically by Tarantool.
-Finally, a space always has an **engine**: *memtx* (default) -- in-memory engine,
-fast but limited in size, or *vinyl* -- on-disk engine for huge data sets.
+..  glossary::
 
-A space is a container for :ref:`tuples <index-box_tuple>`.
-To be functional, it needs to have a :ref:`primary index <index-box_index>`.
-It can also have secondary indexes.
+    tuple
+        A tuple is a group of data values in Tarantool's memory.
+        Think of it as a "database record" or a "row".
```

Comment:
A variant:
"Think of it as an analog of a "database record" or a "row" in a typical SQL database." 
Or maybe without the word "typical" -- "... in SQL databases".
Because we are trying to make a bridge for users who are coming from the SQL world.

Reply:
Doesn't “Think of it as” already mean “it is an analog of”?